### DISCUSSION TITLE
Performance Issues with Rendering Multiple Widgets and tksheet Integration in CustomTkinter

### DISCUSSION THREAD

**Discussion by AnonymousVibrate** [TomSchimansky/CustomTkinter#2747] [Q&A]
Hi! I’d like to ask if this issue is related to CustomTkinter or if my hardware device is struggling to handle simultaneous widget redraws. I’m using a tksheet table, and whenever I click on a row, the data from that row is displayed in 30 Entry/Combobox widgets. However, when I repeatedly click different rows to display their data, I start experiencing noticeable lag after several back-and-forth clicks.

I’m using StringVar() for each Entry/Combobox widget and have tried different approaches to improve performance when updating the variables in these widgets. Additionally, resizing the window (from large to small or vice versa) makes the application even more laggy.

**Comment by FaheemM1020** [comment]
Please prove a sample of reproducible code
